### DISCUSSION TITLE
How to get the original PS4 kernel image (not a memory dump)?

### DISCUSSION THREAD

**Discussion by ultimaweapon** [obhq/obliteration#273] [Q&A]
Not sure if this is even possible. The problem with memory dumps is it is already patched when jailbreaking. Some pointers also mess up, which cannot follow its reference.

**Comment by ultimaweapon** [comment]
@VocalFan can you help on this? I saw you have some contact with the scene not sure if you can ask them to see if this is possible.

**Reply by SoffiMeower** [reply]
I asked them, waiting for reply.

**Reply by SoffiMeower** [reply]
You'd need PUP keys. @ultimaweapon 

- Or another way to decrypt the image.

**Reply by SoffiMeower** [reply]
Oh, it's impossible. Even if you had the decryption keys, you'd need a hardware flasher/dumper to get the kernel image.

**Comment by SoffiMeower** [comment]
You would require the PUP decryption keys, and a hardware flasher/dumper to get the kernel off the device.

**Reply by ultimaweapon** [reply]
Does that mean it need some soldering? If yes are there any instructions for this?

**Reply by SoffiMeower** [reply]
Ahem... So, the kernel is encrypted in the CoreOS blob of SFlash.
Which... Is done with per-console keys.

**Reply by SoffiMeower** [reply]
https://betterwayelectronics.com.au/sce_syscon.html

**Reply by ultimaweapon** [reply]
So the dump from the above tool contains the encrypted kernel? Is it possible to get the key from the console?

**Reply by SoffiMeower** [reply]
Err... Not that they or we know of yet.

**Reply by ultimaweapon** [reply]
So the conclusion is it is impossible to get the decrypted kernel image for now?

**Reply by ultimaweapon** [reply]
Just wondering why we cannot use the same method for decrypting the PUP file with the kernel despite it using the same key.

**Reply by ATouhou** [reply]
Please be careful in case you installed any software from the sex offender hacker (betterwayelectronics) (for more details https://www.youtube.com/watch?v=qFe5LiACN9k )

**Reply by SoffiMeower** [reply]
I... Assume this is probably a spam comment? ...I love Louis Rossman as much as most nerds, but that is a 7 month old video and not relevant to this repo.

**Reply by ATouhou** [reply]
Not a SPAM comment.... In case his software contains a R.A.T.
